user-guides/messaging/README.md

```diff
@@ -108,6 +108,15 @@ Graphical emojis can be enabled/disabled in [Your Account](../managing-your-acco
 
 When graphical emojis are disabled, emoji keywords and shortcuts will be displayed as is  e.g. `:smile:` or `:)`.
 
+## Formatting
+
+To format properly your message you can wrap the text you want to augment with special characters:
+
+* Bold: &#42;Lorem ipsum dolor&#42;;
+* Italic: &#95;Lorem ipsum dolor&#95;;
+* Strike: &#126;Lorem ipsum dolor&#126;;
+* Inline code: &#96;Lorem ipsum dolor&#96;;
+
```

Comment:
You should also add information about link formatting too. Example there are 3 ways.

1. [Text]&#40;http://link)
2. ![alt text]&#40;http://image url)
3. &#60;http://link|Text>

Reply:
Updated.
I've inserted the first one and the third, but I'm not sure about the second.

e.g. `![Alt text](https://purepng.com/public/uploads/large/purepng.com-mario-runningmariofictional-charactervideo-gamefranchisenintendodesigner-1701528632710brm3o.png`) doesn't work on the webapp.

Reply:
no worries, issue has been opened for it. Once others acknowledge it, fix will be made. See here [issue13606](https://github.com/RocketChat/Rocket.Chat/issues/13606).

Reply:
Done

Reply:
Can someone add how to change the font size for a message?   Sorry if this is the wrong place to ask, I'm very new to this.